add doc for v5.1: Deploy -> Operate TiKV (#232)
tikv/website: content/docs/5.1/deploy/operate/maintain.md

---
title: TiUP Common Operations
description: Learn the common operations to operate and maintain a TiKV cluster using TiUP
menu:
    "5.1":
        parent: Operate TiKV
        weight: 3
---

This document describes the following common operations when you operate and maintain a TiKV cluster using TiUP.

- [View the cluster list](./#view-the-cluster-list)
- [Start the cluster](./#start-the-cluster)
- [View the cluster status](./#view-the-cluster-status)
- [Modify the configuration](./#modify-the-configuration)
- [Rename the cluster](./#rename-the-cluster)
- [Stop the cluster](./#stop-the-cluster)
- [Clean up cluster data](./#clean-up-cluster-data)
- [Destroy the cluster](./#destroy-the-cluster)

## View the cluster list

> You can manage multiple TiKV clusters using the TiUP cluster component. When a TiKV cluster is deployed, the cluster appears in the TiUP cluster list.
> 
> To view the list, run the following command:

Reviewer: ```suggestion
You can manage multiple TiKV clusters with the TiUP cluster component.

To view all the deployed TiKV clusters, run the following command:
```

```shell
tiup cluster list
```

## Start the cluster

> The components in the TiKV cluster are started in the following order (The monitoring component is also started):

Reviewer: when will the monitoring components start?

Author: yes, I will add the missing components.

Author: yes, TiUP will start monitoring components after tikv. 
But monitoring components' start order is not a `Must` (Monitoring components can be started before TiKV and PD), so it is not mentioned here.

Author: @zz-jason PTAL https://github.com/tikv/website/pull/232/commits/84a0afc3cb715320ecdc30e30340911122c5049e

**PD -> TiKV**

To start the cluster, run the following command:

```shell
tiup cluster start ${cluster-name}
```

You can start only some of the components by adding the `-R` or `-N` parameters in the command. For example:

- This command starts only the PD component:

    ```shell
    tiup cluster start ${cluster-name} -R pd
    ```

- This command starts only the PD components on the `1.2.3.4` and `1.2.3.5` hosts:

    ```shell
    tiup cluster start ${cluster-name} -N 1.2.3.4:2379,1.2.3.5:2379
    ```

{{< info >}}

> If you start the specified component by using the `-R` or `-N` parameters, make sure the starting order is correct. For example, start the PD component before the TiKV component. Otherwise, the start might fail.

Reviewer 2: ```suggestion
If you start the components with `-R` or `-N` parameters, make sure the order of components is correct. For example, start the PD component before the TiKV component. Otherwise, the start might fail.
```

{{< /info >}}

## View the cluster status

> After starting the cluster, check the status of each component to ensure that they work normally. TiUP provides the `display` command, so you do not have to log in to every machine to view the component status.

Reviewer 2: ```suggestion
After starting the cluster, check the status of each component to ensure that they are up and running. TiUP provides a `display` command to do so, and you don't have to log in to every machine to view the component status.
```
